README.md

```diff
@@ -72,7 +72,7 @@ foreach (var stat in varusRanked.Stats)
 }
 ```
 
-You can find a list of all the available operations in [IRiotApi](RiotSharp/IRiotApi.cs).
+You can find a list of all the available operations in [TournamentApi in Documentation](http://benfradet.github.io/RiotSharp/api/RiotSharp.RiotApi.html).
```

Comment:
very minor but could you replace

> TournamentApi in Documentation

by

> TournamentApi in the documentation

?

Same goes for the links below.